src/Chapter1.hs

```diff
@@ -490,7 +491,9 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n | n < 0 = lastDigit $ abs n
+lastDigit n         = mod n 10
```

Comment:
You don't need to compare with `0` separately. You can always call `abs` because it will keep positive numbers positive